src/features/MovieDetails/Header/styled.js

```diff
@@ -0,0 +1,118 @@
+import styled from "styled-components";
+import poster from "./poster.png";
+
+export const WrapperPoster = styled.div`
+margin: 0 auto;
+    max-width: 1920px;
+    max-height: 769px;
+    height: 50vw;
+    background-image: url("${poster}");
+    background-repeat: no-repeat;
+    background-position: center;;
+    background-size: cover;
+
+    @media ( max-width: ${({ theme }) => theme.breakpoints.medium}px) {
+        max-height:600px;
+};
+
+    @media ( max-width: ${({ theme }) => theme.breakpoints.small}px) {
+        max-height: 400px;
+    };
+`;
+
+export const Wrapper = styled.div`
+    max-width: 1920px;
+    width: 100vw;
+    max-height: 769px;
+    height: 50vw;
```

Comment:
There is something to change here or in the WrapperPoster. The black background is supposed to be the full width of the screen, so this Wrapper is probably the problem.